### ISSUE TITLE
pi-blaster does not relaunched automatically on reboot

### ISSUE THREAD

**Issue by bchr02** [sarfata/pi-blaster#57]
The README.md states:
>To start pi-blaster and have it relaunched automatically on every reboot:
sudo make install

but after doing ```sudo make install``` it does not start the pi-blaster service or start it on reboot. I have to manually start it by typing ```sudo systemctl start pi-blaster```

**Comment by hosiet** [comment]
By reading current `Makefile.am`, you will find that the documentation is outdated and autostart is not true for systems using systemd. You must run `sudo systemctl enable pi-blaster.service` manually after installation.

**Comment by bchr02** [comment]
Thank you! I will try this.

**Comment by bchr02** [comment]
@hosiet  This worked. Thank you. You mentioned the docs were outdated, is pi-blaster not being maintained anymore?

**Comment by sarfata** [comment]
Yes it is still maintained!

If one of you could propose a pull-request to fix this issue, I would gladly merge it. I think the best way to fix this might be to update the Makefile.am to do what the README says (configure program to autostart).

**Comment by bchr02** [comment]
@sarfata Unfortunately, I am not familiar with how Makefile.am works. @hosiet any chance you can do this?

is it just a matter of changing:
````
systemdsystemunit_DATA = pi-blaster.service
````

to:
````
systemdsystemunit_DATA = enable pi-blaster.service
````

**Comment by sarfata** [comment]
I think it's going to be more complicated. #53 is also doing a bunch of related changes so we should wait until it lands before fixing this.

**Comment by sarfata** [comment]
closed by #64
